index.html

```diff
@@ -106,71 +106,109 @@
+          tracks they capture also change.
+        </p>
+        <p>
+          A captured <code>MediaStreamTrack</code> ends
+          when <a href="http://www.w3.org/TR/html5/embedded-content-0.html#ended-playback">playback
+          ends</a> (and the <code>ended</code> event fires) or when the track
+          that it captures is no longer selected or enabled for playback.  A
+          track is no longer selected or enabled if the source is changed by
+          setting the <code>src</code> or <code>srcObject</code> attributes of
+          the media element.
         </p>
         <p>
-          If the selected <a>VideoTrack</a> changes or enabled
-          <a>AudioTrack</a>s change for the media element,
           <code>MediaStreamTrack</code>s are added or removed as necessary to
-          ensure that the <code>MediaStreamTrack</code>s in the
-          <code>MediaStream</code> correctly reflect the changes.  Necessary
-          <code><a
-          href="https://w3c.github.io/mediacapture-main/#event-mediastream-addtrack">addtrack</a></code>
-          and <code><a
-          href="https://w3c.github.io/mediacapture-main/#event-mediastream-removetrack">removetrack</a></code>
-          events are generated to notify applications of these changes.
+          ensure that the state of the captured the <code>MediaStream</code>
+          always correctly reflects changes in the media element. If the
+          selected <a>VideoTrack</a> or set of enabled <a>AudioTrack</a>s for
+          the media element change,
+          a <code><a href="https://w3c.github.io/mediacapture-main/#event-mediastream-addtrack">addtrack</a></code>
+          event with a new <code>MediaStreamTrack</code> is generated for
+          each <a>track</a> that was not previously selected or enabled; and
+          a <code><a href="https://w3c.github.io/mediacapture-main/#event-mediastream-removetrack">removetrack</a></code>
+          events is generated for each <a>track</a> that ceases to be selected
+          or enabled.  A <code>MediaStreamTrack</code> MUST end prior to being
+          removed from the <code>MediaStream</code>.
+        </p>
+        <p>
+          Since a <code>MediaStreamTrack</code> can only end once, a track that
+          is enabled, disabled and re-enabled will be captured as two separate
+          tracks.  Similarly, restarting playback after playback ends causes a
+          new set of captured <code>MediaStreamTrack</code> instances to be
+          created.  Seeking during playback without changing track selection
+          does not generate events or cause a
+          captured <code>MediaStreamTrack</code> to end.
+        </p>
+        <p>
+          At any time, a media element might not have active content available
+          for capture on a given track for a variety of reasons:
         </p>
+        <ul>
+          <li>
+            Media playback could be paused or ended.
```

Comment:
If playback ended then the MediaStreamTrack has ended and been removed, per above.